src/meta/src/barrier/command.rs

```diff
@@ -68,6 +71,9 @@ pub struct Reschedule {
 
     /// Reassigned splits for source actors
     pub actor_splits: HashMap<ActorId, Vec<SplitImpl>>,
+
+    /// Whether this fragment is injectable.
+    pub injectable: bool,
```

Comment:
Can you elaborate more in doc? When it'll be not?

Reply:
The `injectable` means whether the fragment contains any executors that are able to receive barrier. It's required to be in the to send list for barrier injection. I will add more comments for it.